src/routers/OpenRouter.js

```diff
@@ -1,22 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { Route } from 'react-router-dom';
-
-const OpenRouter = ({ component: Component, path, previewType }) => (
```

Comment:
whe we delete this ?

Reply:
It is same as Route component, we don't need this